apps/block_scout_web/lib/block_scout_web/controllers/api/v2/import_controller.ex

```diff
@@ -96,6 +92,32 @@ defmodule BlockScoutWeb.API.V2.ImportController do
     end
   end
 
+  def delete_token_info(
+        conn,
+        %{
+          "token_address_hash" => token_address_hash_string
+        } = params
+      ) do
+    with {:ok, token} <- validate_api_key_address_hash_and_token(token_address_hash_string, params["api_key"]) do
```

Comment:
nitpick

```suggestion
          "token_address_hash" => token_address_hash_string,
          "api_key" => api_key
        }
      ) do
    with {:ok, token} <- validate_api_key_address_hash_and_token(token_address_hash_string, api_key) do
```

same for `import_token_info/2`

Reply:
If I will apply your suggestion, then user will receive 422 error without any explanations on the request without api_key param. We already have logic on check api_key, in `validate_api_key_address_hash_and_token/2`. So current way is the intentional, to not to match `"api_key" => api_key`

Reply:
Oh, I see, didn't spot that. Thanks